### DISCUSSION TITLE
How to show prefix right before note title/name in the parent note view of all child notes

### DISCUSSION THREAD

**Discussion by MotikaCraft-TheJoshua** [TriliumNext/Trilium#4362] [Q&A]
I hope Discussion title is enough to know what i am asking

![image](https://github.com/zadam/trilium/assets/56804505/832cbeaa-b1e5-4f28-aa8e-f99bf59d0999)

**Comment by zadam** [comment]
Hi, this one is valid, fixed in master which will become 0.62

**Comment by MotikaCraft-TheJoshua** [comment]
> Hi, this one is valid, fixed in master which will become 0.62

Hi,
thank you so much for this!

I am very sorry if this is out of scope to ask here, but I am very new to Trilium Notes, yet, i fell in love with it so much!

I am using docker image to selfhost it, of course

Can you tell me, what would be the process of upgrading to next release, because you mentioned this will be implemented in next (`0.62`) build:
- is it safe to just pull latest image and use instead of current one, or would it be more safe to just build from very latest (when `0.62` is out) a fresh server/container and then re-import notes i already have in my current build (i'm on `0.60.4`)?
- Also, how safe is upgrading in terms on how notes are handled, can something get broken? I know it is for everything recommended to backup, but i still would be happy and glad to hear what you can say as you developed thi.

Thank you so much if you answer me this, i greatly appreciate it...!

**Reply by jwhonce** [reply]
Using containers, I run a 0.60.4 and a 0.61.latest-beta server on different ports. I copy the 0.60.4 backup/backup-now.db to the 0.61.latest-beta document.db and start the 0.61.latest-beta container which does the database upgrade as it starts up. This let's me test and see the latest goodies while not breaking my daily driver. 

Trilium has become integral to my work day.

**Reply by zadam** [reply]
Normal release upgrades are rather safe, routine operations which don't threaten the content.

The 0.60-0.61 is a major upgrade (such won't happen for a long time again) and therefore the risks involved are somewhat larger. Trilium automatically makes a backup before migrating the database, though.

I recommend using now 0.60.X or 0.61.X.

0.62 is not released in any form, it's just master branch in git and there may be bugs. Having said that, the changes in 0.62 are mostly UX and using it even now likely brings almost no risk to data.
